.env.sample

```
@@ -0,0 +1,10 @@
+TYPEORM_TYPE=mysql
```

dbmate를 통해 데이터베이스에 스키마를 생성하기 위한 `DATABASE_URL` 정보도 추가해주세요!
```suggestion
DATABASE_URL="rdbms://username:password@host:port/database-name"

TYPEORM_TYPE=mysql
```